src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/config.html.twig

```diff
@@ -137,33 +143,39 @@
         {% endif %}
     </div>
 
-    {% set symfony_status = { dev: 'Unstable Version', stable: 'Stable Version', eom: 'Maintenance Ended', eol: 'Version Expired' } %}
-    {% set symfony_status_class = { dev: 'warning', stable: 'success', eom: 'warning', eol: 'error' } %}
-    <table>
-        <thead class="small">
-            <tr>
-                <th>Symfony Status</th>
-                <th>Bugs {{ collector.symfonystate in ['eom', 'eol'] ? 'were' : 'are' }} fixed until</th>
-                <th>Security issues {{ collector.symfonystate == 'eol' ? 'were' : 'are' }} fixed until</th>
-                <th></th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td class="font-normal">
-                    <span class="label status-{{ symfony_status_class[collector.symfonystate] }}">{{ symfony_status[collector.symfonystate]|upper }}</span>
-                    {% if collector.symfonylts %}
-                        &nbsp; <span class="label status-success">Long-Term Support</span>
-                    {% endif %}
-                </td>
-                <td class="font-normal">{{ collector.symfonyeom }}</td>
-                <td class="font-normal">{{ collector.symfonyeol }}</td>
-                <td class="font-normal">
-                    <a href="https://symfony.com/releases/{{ collector.symfonyminorversion }}#release-checker">View roadmap</a>
-                </td>
-            </tr>
-        </tbody>
-    </table>
+    {% set symfony_status = { dev: 'In Development', stable: 'Maintained', eom: 'Unmaintained', eol: 'Unmaintained' } %}
```

Comment:
shouldn't we have different wordings for eom and eol ? An LTS version reaching eom is not unmaintained yet. It still receives security fixes (while it does not after eol)